tests/smb-version-keyword-invalid/test.yaml

```diff
@@ -0,0 +1,14 @@
+requires:
+  min-version: 7
```

Comment:
Please make it 8

Reply:
Is it okay to force push that update or should I make a new PR?

Reply:
For me it is ok, but maybe better to do a new PR ;-)

Reply:
updated